.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+before_install:
+    - brew update
+install:
+    - mkdir -p $(brew --repo)/Library/Taps/travis
+    - ln -s $PWD $(brew --repo)/Library/Taps/travis/homebrew-moira
+    - brew tap --repair
+script:
+    - brew audit -v $PACKAGE
+    - brew install -v $PACKAGE
+    - brew test -v $PACKAGE    
+matrix:
+    include:
+        - env: macos=10.13 PACKAGE=moira
```

Comment:
This is not 10.13.

From https://docs.travis-ci.com/user/reference/osx/#OS-X-Version, xcode9.1 provides Xcode9.1 on OS X 10.12.

It may be better to punt the idea of being able to test on High Sierra just yet.

We should test all versions of xcode that's available per that reference. Might as well punt all the `env` key-values for each dict member of the `include` list, or just keep them as a reference for which macOS/OS X version is being used for each `osx_image` (and add a comment saying that).

Reply:
I sent Travis CIs support team mail asking what's up with that. We don't need to test with every version of Xcode, just the most recent of each major version, which is at this point 6.4, 7.3.1, 8.3.3, and 9.1

Reply:
Okay, that's reasonable.

Reply:
In 4ade5f4, I updated the macos variable for Xcode 9.3 to say 10.12 for now